language/English/strings.po

```diff
@@ -16254,3 +16254,12 @@ msgstr ""
 msgctxt "#38010"
 msgid "GPU accelerated"
 msgstr ""
+
+#: system/settings/settings.xml
+msgctxt "#38011"
+msgid "Hide All Items entry"
```

Comment:
Maybe add (escaped) double-quotes around `All Items` to make it more obvious. (same in the description).